lib/engines/shopping_cart/app/controllers/shopping_cart/application_controller.rb

```diff
@@ -0,0 +1,18 @@
+module ShoppingCart
+  class ApplicationController < ::ApplicationController
+    protect_from_forgery with: :exception
```

Comment:
@b-artem your engine should not override protection strategy

Reply:
@Dulenchik Thank you, corrected it